From 21524598110e7a55113205dda4bc1a6d5987111c Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 9 May 2017 06:41:33 +0000
Subject: [PATCH] change order of detecting libegl and libgles2

This actually helps building it on rpi/userland
where libEGL depends on few functions from libGLESv2
see
https://www.raspberrypi.org/forums/viewtopic.php?t=7090

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index b9e4240..1ca0297 100644
--- a/configure.ac
+++ b/configure.ac
@@ -951,8 +951,8 @@ if test "$use_gles" = "yes"; then
       AC_MSG_RESULT(== WARNING: OpenGLES support is assumed.)
       LIBS="$LIBS -lEGL -lGLESv2 -lbcm_host -lvcos -lvchiq_arm -lmmal -lmmal_core -lmmal_util -lvcsm"
     else
-      AC_CHECK_LIB([EGL],   [main],, AC_MSG_ERROR($missing_library))
       AC_CHECK_LIB([GLESv2],[main],, AC_MSG_ERROR($missing_library))
+      AC_CHECK_LIB([EGL],   [main],, AC_MSG_ERROR($missing_library))
     fi
   fi
 else
-- 
1.9.1

